doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -42,7 +42,7 @@ These updates appear in detail in the sections below. Many also appear in
 
```

Comment:
Can you add a section under CLM5 updates to highlight new features after CLM5? This can be 26.1.1 and titled something like: “Available new features since the CLM5 release” and can include a bullet that highlights the addition of bioenergy crops. This will help us keep track of what was included in the CLM5 release and allow us to highlight the new work available for the next release.